src/core_plugins/elasticsearch/lib/kibana_index_mappings.js

```diff
@@ -1,16 +1,24 @@
-export const mappings = {
+const currentMappings = {
```

Comment:
We need to keep an instance of the modified mappings per server, not in the module scope